hiring/remote_test/README.md

```diff
@@ -0,0 +1,16 @@
+# Remote test
+
+This is our current remote test for screening candidates. We provide a candidate with both a brief and a marking guide that clearly explains what we are looking for technically. The purpose of this test is to screen out candidates that cannot complete a set of instructions and perform tasks within a rails app.
+
+In order to proceed to the next step, a candidate must achieve 80% marks or more.
+
+ - [Brief](brief.md)
+ - [Marking Guide](marking_guide.md)
+
+A few things to note:
+
+ - You can take as much time to complete this test
```

Comment:
(though I understand it from a being fair to people to haven't got much free-time perspective)